extensions/blocks/story/story.php

```diff
@@ -218,9 +232,14 @@ function render_slide( $media, $index = 0 ) {
 			$media_template = render_image( $media, $index );
 			break;
 		case 'video':
-		case 'file':
 			$media_template = render_video( $media, $index );
 			break;
+		case 'file':
+			// VideoPress videos on WordPress.com can have type 'file'.
```

Comment:
Do you think you could expand this comment a bit, maybe with an example, so a few months from now anyone can quickly understand what values we expect from `$media['mime']` here, on WordPress.com and in Jetpack?

Reply:
So I haven't been able to reproduce the original issue I was seeing, even with videos/posts I'm pretty sure were showing `type=file` before on WordPress.com. All VideoPress videos I try on WP.com sites now have `type=video`.

However I do see some cases of `type=file` for VideoPress videos on Jetpack sites. I'm not sure what the ultimate cause is, it doesn't seem super common. (I know @Tug ran into this as well which is why he added the `file` case in https://github.com/Automattic/jetpack/pull/17708 in the first place.)

For now I updated the comment and added the `mime` cases I've seen so far to match, let me know what you think.